NeuroID/NeuroIDClass/Extensions/NIDSession.swift

```diff
@@ -206,4 +208,37 @@ public extension NeuroID {
 
         return true
     }
+
+    static func startAppFlow(siteID: String, userID: String? = nil) -> SessionStartResult {
+        if NeuroID.clientKey == nil || NeuroID.clientKey == "" {
+            NIDLog.e("Missing Client Key - please call configure prior to calling start")
+            return SessionStartResult(false, "")
+        }
+
+        // immediately flush events before anything else
+        groupAndPOST(forceSend: true)
```

Comment:
Is this still okay if we are in a `stopped` state? 
I agree queue should be flushed before anything else but just wanted to check if it was safe to do if `stopped`

Reply:
Yes I think even if we are in a "stopped" state meaning we are collecting anymore we should still flush the events. This also fixes a very minor race condition when stop/stopSession is called where we send the packet of events and then capture a CLOSE_SESSION event but never send it